src/components/molecules/NavBar/components/NavBarRadio/NavBarRadio.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useMemo, useRef, useState } from "react";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import classNames from "classnames";
+
+import {
+  currentVenueSelectorData,
+  radioStationsSelector,
+} from "utils/selectors";
+import { hasElements } from "utils/types";
+
+import { useSelector } from "hooks/useSelector";
+import { useRadio } from "hooks/useRadio";
+
+import { RadioModal } from "components/organisms/RadioModal/RadioModal";
+
+import "./NavBarRadio.scss";
+
+export const NavBarRadio: React.FC = () => {
+  const venue = useSelector(currentVenueSelectorData);
+
+  const radioStations = useSelector(radioStationsSelector) ?? [];
+  const [isRadioPlaying, setIsRadioPlaying] = useState(false);
+  const hasRadioStations = radioStations && radioStations.length;
+  const radioStation = !!hasRadioStations && radioStations[0];
+  const isSoundCloud =
+    !!hasRadioStations && RegExp("soundcloud").test(radioStations[0]);
+
+  const sound = useMemo(
+    () =>
+      radioStations && hasElements(radioStations) && !isSoundCloud
+        ? new Audio(radioStations[0])
+        : undefined,
+    [isSoundCloud, radioStations]
+  );
+  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
+
+  const [showRadioPopover, setShowRadioPopover] = useState(false);
+
+  const toggleShowRadioPopover = useCallback(
+    () => setShowRadioPopover((prevState) => !prevState),
+    []
+  );
+  const { volume, setVolume } = useRadio(isRadioPlaying, sound);
+  const showNormalRadio = (venue?.showRadio && !isSoundCloud) ?? false;
+  const showSoundCloudRadio = (venue?.showRadio && isSoundCloud) ?? false;
+
+  const radioFirstPlayStateLoaded = useRef(false);
+  const showRadioOverlay = useMemo(() => {
+    if (!radioFirstPlayStateLoaded.current) {
+      const radioFirstPlayStorageKey = "radioFirstPlay";
+      const radioFirstPlayState = localStorage.getItem(
+        radioFirstPlayStorageKey
+      );
+      if (!radioFirstPlayState) {
+        localStorage.setItem(radioFirstPlayStorageKey, JSON.stringify(true));
```

Comment:
![image](https://user-images.githubusercontent.com/753891/117748310-189e7680-b253-11eb-857b-6a8a55fe47da.png)
```suggestion
        localStorage.setItem(radioFirstPlayStorageKey, "true");
```